tests/ballerina-test-integration/src/test/resources/grpc/invalid_resource_client.bal

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/io;
+import ballerina/grpc;
+
+endpoint HelloWorldBlockingClient helloWorldBlockingEp {
+    url:"http://localhost:9090"
+};
+
+
```

Comment:
Extra new line??

Reply:
Fixed in b63fe01732a08a31f3290e5d72f1528999e2e999